holoviews/plotting/bokeh/util.py

```diff
@@ -193,6 +163,39 @@ def get_ids(obj):
     return list(itertools.chain(*ids))
 
 
+def replace_models(obj):
+    """
+    Processes references replacing Model and HasProps objects.
+    """
```

Comment:
Maybe 'Recursively processes references, replacing Models with there .ref values and HasProps objects with their property values'. At least that is how I understand the code!